components/grid/filter/overview.md

```diff
@@ -37,16 +37,17 @@ The filter menu can display a [list of checkboxes]({%slug grid-checklist-filter%
 
 ## Filter Descriptors
 
-The Grid filter state is stored in [`CompositeFilterDescriptors`](/blazor-ui/api/Telerik.DataSource.CompositeFilterDescriptor). Use the following information if you want to [get or change the Grid filters programmatically]({%slug grid-state%}).
+The filtering criteria for each filtered field is stored in an individual [`CompositeFilterDescriptor`]({%slug common-features-composite-filter-descriptor%}). Use the following information if you want to [get or change the Grid filters programmatically]({%slug grid-state%}).
```

Comment:
What do you mean by "Use the following information if you want to...?" - do you use this expression to introduce the linked resource or the paragraphs that follow? If you mean the linked resource, please rewrite this sentence as it's unclear and confusing.
```suggestion
The filtering criteria for each filtered field are stored in an individual [`CompositeFilterDescriptor`]({%slug common-features-composite-filter-descriptor%}). Use the following information if you want to [get or change the Grid filters programmatically]({%slug grid-state%}).
```